github/ci/prow/files/jobs/ovs-cni/ovs-cni-presubmits.yaml

```diff
@@ -0,0 +1,34 @@
+---
+presubmits:
+  kubevirt/ovs-cni:
+    - name: pull-ovs-cni-check-pr
```

Comment:
`pull` and `pr` are a duplication in this case. `pull` is the prefix for jobs for pull-requests. Normally we end them with something like `test`, `buil`, ... instead of `check`.

Reply:
Yeah, I know that but I didn't really know what to put in the name here...
maybe just `pull-ovs-cni`?

Reply:
what does it do? unit-tests? Then `-test`.

Reply:
`pull-ovs-cni-test`

Reply:
but `pull-ovs-cni` works too for me.

Reply:
@phoracek up to you

Reply:
It runs functional tests on local cluster. `pull-ovs-cni` is good

Reply:
updated

Reply:
for functional tests we typically call them `pull-e2e-*`.

Reply:
Once (if) we move unit tests to prow as well, we can simply do the renaming. Not a big deal for me, we can change it to e2e or keep as is, both work for me

Reply:
updated again :+1: I hope it's the last time for this PR :)